ephios/api/urls.py

```diff
@@ -15,11 +15,23 @@
     ApplicationUpdate,
 )
 from ephios.api.views.events import EventViewSet
-from ephios.api.views.users import UserProfileMeView
+from ephios.api.views.users import (
+    MailBasedUserParticipationView,
+    UserParticipationView,
+    UserProfileMeView,
+)
 from ephios.extra.permissions import staff_required
 
 router = routers.DefaultRouter()
 router.register(r"events", EventViewSet)
+router.register(
+    r"users/(?P<user>[\d]+)/participations", UserParticipationView, basename="user-participations"
```

Comment:
I think we should create a "/users/" endpoint as well if we have the participations endpoint. Should be easy.